### PR TITLE
VP-2065 Fix price blade error when in platform registered more than 20 catalogs

### PR DESCRIPTION
**Pull request by AliveMen**
Change search criteria to get all catalogues and reduce response group.

### CODE REVIEW

File: src/VirtoCommerce.CatalogModule.Web/Controllers/Api/CatalogModuleCatalogsController.cs [hunk 1]
```diff
@@ -39,12 +39,13 @@ ICatalogService catalogService
         /// <remarks>Get common and virtual Catalogs list with minimal information included. Returns array of Catalog</remarks>
 		[HttpGet]
         [Route("")]
-        public async Task<ActionResult<Catalog[]>> GetCatalogs(string sort = null, int skip = 0, int take = 20)
+        public async Task<ActionResult<Catalog[]>> GetCatalogs(string sort = null, int skip = 0, int take = 1000)
```
Comment (yecli) [inline src/VirtoCommerce.CatalogModule.Web/Controllers/Api/CatalogModuleCatalogsController.cs:42]: Shouldn't we change that condition on the client side?
I think it is safer to explicitly specify this value on the client if we need more.

File: src/VirtoCommerce.CatalogModule.Web/Controllers/Api/CatalogModuleCatalogsController.cs [hunk 2]
```diff
@@ -39,12 +39,13 @@ ICatalogService catalogService
...
         {
             var criteria = AbstractTypeFactory<CatalogSearchCriteria>.TryCreateInstance();
             criteria.Sort = sort;
             criteria.Skip = skip;
             criteria.Take = take;
+            criteria.ResponseGroup = CatalogResponseGroup.Info.ToString();
```
Comment (yecli) [inline src/VirtoCommerce.CatalogModule.Web/Controllers/Api/CatalogModuleCatalogsController.cs:48]: @tatarincev 
Should we set minimal response group here according to the method description - "Get common and virtual Catalogs list with minimal information included"(as it is Full by default)?

File: src/VirtoCommerce.CatalogModule.Web/Controllers/Api/CatalogModuleCatalogsController.cs (hunk 1 above)
Comment (AliveMen) [reply src/VirtoCommerce.CatalogModule.Web/Controllers/Api/CatalogModuleCatalogsController.cs:44]: reverted

File: src/VirtoCommerce.CatalogModule.Web/Controllers/Api/CatalogModuleCatalogsController.cs (hunk 2 above)
Comment (AliveMen) [reply src/VirtoCommerce.CatalogModule.Web/Controllers/Api/CatalogModuleCatalogsController.cs:50]: reverted

File: src/VirtoCommerce.CatalogModule.Web/Scripts/widgets/dashboard.js [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
 .controller('virtoCommerce.catalogModule.dashboard.catalogsWidgetController', ['$scope', '$state', 'virtoCommerce.catalogModule.catalogs', function ($scope, $state, catalogs) {
     $scope.data = { count: '', descr: 'Catalogs' };
 
     $scope.widgetAction = function () {
         $state.go('workspace.catalog');
     };
 
-    catalogs.getCatalogs({}, function (data) {
-        $scope.data.count = data.length;
+    catalogs.search({take:1000}, function (data) {
```
Comment (yecli) [inline src/VirtoCommerce.CatalogModule.Web/Scripts/widgets/dashboard.js:9]: Here I think we should specify take:0, as we only using length. Please check, possible redundant queries

Comment (AliveMen) [reply src/VirtoCommerce.CatalogModule.Web/Scripts/widgets/dashboard.js]: fixed

**Pull request merged**